rust/src/types/mod.rs

```diff
@@ -1 +1,3 @@
 mod amount;
+
+pub use amount::u256;
```

Comment:
It does not make much sense neither than the namespace is called amount and the type u256. This comes from my own misunderstanding. please rename this module whenever you find time, to something that makes more sense, like u256.